gqueries/output_elements/output_series/waterfall_captured_emssions/negative_captured_bio_emissions_waterfall_captured_emissions.gql

```diff
@@ -0,0 +1,3 @@
+- query = 
+        NEG(Q(emissions_captured_biogenic_total))
+- unit = tonne
```

Comment:
This query uses hardcoded references to power plants. This means that torrefied biomass pellets (and potentially other biogenic) flows are missing. I would rewrite the query to be more generic, somethin like:
`SUM(V(G(electricity_production),"demand * weighted_carrier_potential_co2_per_mj * free_co2_factor"))`

Reply:
I just tested this query, seems to be accurate for what we are searching.